apps/academy/src/pages/_app.tsx

```diff
@@ -100,8 +101,44 @@ type AppPropsWithLayout<P> = AppProps<P> & {
 };
 
 function MyApp({ Component, pageProps }: AppPropsWithLayout<{ session: Session | null }>) {
+  const router = useRouter();
   const getLayout = Component.getLayout ?? ((page) => page);
 
+  // If the current route is '/fundamentals', don't use the Layout
+  if (router.pathname === "/fundamentals") {
+    return (
+      <WagmiConfig config={wagmiConfig}>
+        <SessionProvider refetchInterval={0} session={pageProps.session}>
+          <RainbowKitSiweNextAuthProvider>
+            <RainbowKitProvider
+              chains={chains}
+              initialChain={polygonMumbai}
+              appInfo={{
+                appName: "Developer DAO Academy",
+                learnMoreUrl: "https://academy.developerdao.com",
+              }}
+              theme={{
+                lightMode: academyLightTheme,
+                darkMode: academyDarkTheme,
+              }}
+            >
+              <ThemeProvider attribute="class">
+                <DefaultSeo {...SEO} />
+                <Head>
+                  <meta
+                    name="viewport"
+                    content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no, viewport-fit=cover"
+                  />
+                </Head>
+                <Component {...pageProps} />
+              </ThemeProvider>
+            </RainbowKitProvider>
+          </RainbowKitSiweNextAuthProvider>
+        </SessionProvider>
+      </WagmiConfig>
+    );
+  }
```

Comment:
why don't using the layout? 
The layout includes the connect button and the navigation bar
the layout was made to be used

Reply:
This page layout is split into two sides. The left side is fixed, while the right side needs to scroll.
![CleanShot 2023-10-27 at 08 16 22](https://github.com/Developer-DAO/academy-turbo/assets/57156589/09833168-947d-4003-a670-96cd6aec9fb7)


The default layout doesn't seem to enable that.

Yes, the layout was made to be used, although in this case, didn't seem like the ideal fit to achieve this result of a fixed left side and a scrolling right side per the design.

@peteruche21 can you chime in here to see if I am missing something?

Take a look at the demo to see what I am referring to:
https://academy-turbo-git-fundamentals-developdao.vercel.app/fundamentals

Reply:
> The layout is split into two sides in the fundamental and track page case. The left side is fixed, while the right side needs to scroll. ![CleanShot 2023-10-27 at 08 16 22](https://user-images.githubusercontent.com/57156589/278653738-09833168-947d-4003-a670-96cd6aec9fb7.gif) [ ![CleanShot 2023-10-27 at 08 16 22](https://user-images.githubusercontent.com/57156589/278653738-09833168-947d-4003-a670-96cd6aec9fb7.gif) ](https://user-images.githubusercontent.com/57156589/278653738-09833168-947d-4003-a670-96cd6aec9fb7.gif) [ ](https://user-images.githubusercontent.com/57156589/278653738-09833168-947d-4003-a670-96cd6aec9fb7.gif)
> 
> The default layout doesn't seem to enable that.
> 
> Yes, the layout was made to be used, although in this case, didn't seem like the ideal fit to achieve this result of a fixed left side and a scrolling right side per the design.
> 
> @peteruche21 can you chime in here to see if I am missing something?
> 
> Take a look at the demo to see what I am referring to: https://academy-turbo-git-fundamentals-developdao.vercel.app/fundamentals

I see it, but using the route.path to compare when it's fundamentals page and to not use the layout, is not the way to go. 
a. we are duplicating code
b. we are not generating a layout that can be shareable to other page (homepage and viceversa)

Reply:
I understand. My rationale was to make something that works first, then work on optimization.

I will abstract the layout from the `fundamentals.tsx` page and move it to `_app.tsx`.